src/modules/job-manager/jobtap.c

```diff
@@ -66,6 +66,7 @@ struct jobtap {
     zhashx_t *plugins_byuuid;
     zlistx_t *jobstack;
     char last_error [128];
+    bool configured;
```

Comment:
Very minor typo in commit message: `A conf.update jobtap callbacks is added` -> ` conf.update jobtap callback is added`